src/NodaTime.Web/Markdown/unstable/calendars.md

```diff
@@ -0,0 +1,223 @@
+@Title="Calendars"
+
+As described in the [core concepts documentation](concepts), a calendar system is a scheme
+for dividing time into eras, years, months and days and so on. As a matter of simplification,
+Noda Time treats every day as starting and ending at midnight, despite some calendars (such as
+the Islamic and Hebrew calendars) traditionally having days stretching from sunset to sunset.
```

Comment:
could include the Wondrous calendar, as its days start at sunset too

Reply:
Yup, will do.